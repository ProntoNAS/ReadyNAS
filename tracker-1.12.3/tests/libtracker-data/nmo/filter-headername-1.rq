SELECT ?headerName
WHERE { ?x nmo:headerName ?headerName }
ORDER BY ?headerName
