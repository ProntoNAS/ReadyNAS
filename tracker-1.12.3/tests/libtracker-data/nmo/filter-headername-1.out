"stringly data for nmo:headerName"
